tuxemon/event/actions/start_battle.py

```diff
@@ -68,17 +68,15 @@ def start(self) -> None:
         npc = world.get_entity(self.parameters.npc_slug)
         assert npc
         if len(npc.monsters) == 0:
-            logger.warning("npc has no monsters, won't start")
+            logger.warning(f"npc '{self.parameters.npc_slug}' has no monsters, won't start trainer battle.")
```

Comment:
Minor improvements added in an isolated commit. Not critical to this PR so I can remove them.